website/source/guides/advanced-scheduling/advanced-scheduling.html.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: "guides"
+page_title: "Advanced Scheduling Features"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+    Introduce advanced scheduling features including affinity and spread.
+---
+
+# Advanced Scheduling with Nomad
+
+The Nomad [scheduler][scheduling] uses a bin packing algorithm to optimize the resource utilization and density of applications in your Nomad cluster. This feature, in addition to the ability to restrict which nodes a workload runs on with the [constraint][constraint-stanza] stanza, provides the Nomad operator an efficient and powerful way to schedule jobs.
```

Comment:
I think we should motivate the advanced placement features a bit more. Suggestion: 

```
The Nomad [scheduler][scheduling] uses a bin packing algorithm to optimize the resource utilization and density of applications in your Nomad cluster. 
Operators could limit the set of nodes that allocations landed through the [constraints] stanza and [distinct hosts] for some flexibility around placement. 

Nomad 0.9 release adds new features to allow operators more fine grained control over allocation placement. This enables use cases like:
-  Prefering a certain class of nodes for a specific application, via the [affinity] stanza
-  Spreading allocations across a datacenter or rack

Please refer to the following guides for using affinity and spread in Nomad 0.9 
...
```